### PR TITLE
Added Architecture files

### PR DESCRIPTION
**Pull request by PrashitaK**

### CODE REVIEW

File: Assignment2/Architecture.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+**Age**: 22
+**Occupation**: College student
+**Background**: Lewis is a college student who wants to improve his language skills to enhance his job prospects. He is interested in learning German and uses Duolingo regularly to practice his language skills. He prefers the app because it is convenient and flexible, allowing him to learn at his own pace.
+**Context Diagram** - [Here](https://github.com/SWENGG4Y2023/SWENGG4Y2023Team07/blob/main/Assignment2/ContainerLearner.puml)
+
+
+* **Educator persona**: Lilith 
+**Age**: 22
+**Occupation**: School Teacher
+**Background**: Lilith is a school teacher who wants to use the app as a teaching platform for her students. She believes that learning can be hybrid in a way that she teaches her students Italian in school and they can pratcise their lesson of the day on Duolingo for School. This enables students to pratise in a fun and exiciting way. It makes it easier to grade her students from the platform as well.
```
Comment (srinair7) [inline Assignment2/Architecture.md]: spell checks😂

File: Assignment1/SRS.md [hunk 2]
```diff
@@ -124,6 +127,8 @@ Duolingo is a language-learning platform that provides users with a gamified and
 ### 2.6. User Documentation:
  The Duolingo application provides user documentation in the form of a help centers, user guides, and a community forum. The help center provides answers to frequently asked questions, while the user guides provide step-by-step instructions on how to use the app. The community forum allows users to connect with other language learners, ask questions, and share tips and strategies.
 
+ The latest feature added to Duolingo are the podcasts language enthusiasts can hear to anywhere, everywhere.
```
Comment (srinair7) [inline Assignment1/SRS.md]: spell checks

**Review by srinair7 — CHANGES_REQUESTED** [review]
Some corrections here and there. Other than that LGTM!

**Comment by PrashitaK** [comment]
@srinair7 @vatsla20 @keerthigpi @iamsebinsunny 
Someone review this once naa.

**Review by keerthigpi — APPROVED** [review]
Good job!💛

**Review by vatsla20 — APPROVED** [review]
Looks great

**Pull request merged**